tests/console/docker_runc.pm

```diff
@@ -48,7 +55,7 @@ sub run {
     record_info 'Test #1', 'Test: Installation';
     if (is_caasp('DVD') && !check_var('SYSTEM_ROLE', 'plain')) {
```

Comment:
please remove "check_var('SYSTEM_ROLE', 'plain')" - otherwise you will run "zypper in" on caasp in else block